pkg/ui/ccl/src/views/clusterviz/containers/map/circleLayout.tsx

```diff
@@ -14,13 +14,16 @@ import { getChildLocalities } from "src/util/localities";
 import { LocalityView } from "./localityView";
 import { NodeView } from "./nodeView";
 import { LivenessStatus } from "src/redux/nodes";
+import { cockroach } from "src/js/protos";
+import Liveness = cockroach.storage.Liveness;
```

Comment:
should this be `import`?  I'm surprised that even works?

Reply:
Changed some of them to `type`, but for `NodeLivenessStatus`, it has to be `import`, otherwise the enum members are not available. (in this file: https://github.com/cockroachdb/cockroach/pull/23255/files#diff-85a6b40255d272377fc904208ce0f4ddR25)